src/routes/choice-exercises/WordChoiceExerciseScreen.tsx

```diff
@@ -16,7 +16,7 @@ interface WordChoiceExerciseScreenPropsType {
 const WordChoiceExerciseScreen = ({ navigation, route }: WordChoiceExerciseScreenPropsType): ReactElement | null => {
   const { extraParams } = route.params
   const { trainingSetId } = extraParams
-  const { data: documents, loading } = useLoadDocuments(trainingSetId)
+  const { data: documents, loading } = useLoadDocumentsRandomOrder(trainingSetId)
```

Comment:
Wouldn't it be simpler to just apply the shuffle function here instead of introducing a new hook?
Maybe the useLoadDocuments hook could also just take a paramter `shuffle`